action.yml

```diff
@@ -5,10 +5,30 @@ branding:
   icon: "at-sign"
   color: "green"
 inputs:
-  GITHUB_TOKEN:
-    description: "GitHub token"
-    required: true
-
+  actions:
+    description: "Actions to correct"
+    required: false
+    default: ".github/workflows"
+  dockerfiles:
+    description: "Dockerfiles to correct"
+    required: false
+    default: "Dockerfile"
+  kubernetes:
+    description: "Kubernetes manifests to correct"
+    required: false
+    default: ""
+  docker_compose:
+    description: "Docker Compose files to correct"
+    required: false
+    default: ""
+  open_pr:
```

Comment:
what would happen if open_pr is false, I guess it would never work?

Reply:
If open_pr is false it would run the action, parse the files, and output what it did, perhaps fail the run if `fail_on_unpinned` is set, but it won't create a PR.